client/src/pages/Feed.js

```diff
@@ -108,16 +280,60 @@ const Feed = () => {
         handleFilterModal,
         handleQuit,
         handleLocation,
+        handleOnClose,
+        showFilters,
       }}
     >
       <FeedWrapper>
-        <FilterBox />
-        <Posts filteredPosts={fakePosts} />
-        <SvgIcon
-          src={creatPost}
-          onClick={handleCreatePost}
-          className="create-post"
-        />
+        <LayoutWrapper>
+          <SiderWrapper
+            breakpoint="md"
+            className="site-layout-background"
+            width={290}>
+            <div>
+              <MenuWrapper
+                defaultSelectedKeys={['ALL']}
+                onClick={handleChangeType}
+              >
+                {Object.keys(HELP_TYPE).map((item, index) =>
+                  <Menu.Item key={item}>
+                    {HELP_TYPE[item]}
+                  </Menu.Item>
+                )}
+              </MenuWrapper>
+              <FiltersWrapper>
```

Comment:
We need to show the selected tags as well (maybe we can work on this on another PR)

<img width="208" alt="Screen Shot 2020-05-10 at 9 14 14 PM" src="https://user-images.githubusercontent.com/42228386/81523702-8b54d900-9303-11ea-938c-0525426fb2b7.png">

Reply:
FiltersList Component shows selected tags.